powerauth-restful-model/src/main/java/io/getlime/security/powerauth/rest/api/model/entity/ActivationRecovery.java

```diff
@@ -19,45 +19,24 @@
  */
 package io.getlime.security.powerauth.rest.api.model.entity;
 
+import lombok.Data;
+
 /**
  * Activation recovery model class for recovery code and PUK.
  *
  * @author Roman Strobl, roman.strobl@wultra.com
  */
+@Data
 public class ActivationRecovery {
 
-    private String recoveryCode;
-    private String puk;
-
     /**
-     * Get recovery code.
-     * @return Recovery code.
+     * Recovery code.
      */
-    public String getRecoveryCode() {
-        return recoveryCode;
-    }
-
-    /**
-     * Set recovery code.
-     * @param recoveryCode Recovery code.
-     */
-    public void setRecoveryCode(String recoveryCode) {
-        this.recoveryCode = recoveryCode;
-    }
+    private String recoveryCode;
 
     /**
-     * Get recovery PUK.
-     * @return Recovery PUK.
+     * Recovery PUK.
      */
-    public String getPuk() {
-        return puk;
-    }
+    private String puk;
```

Comment:
I suggest we exclude this field from `toString` to avoid logging the puk by mistake.